tasks.schema.json

```diff
@@ -0,0 +1,329 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Why are you adding the tasks schema?

Reply:
Left over, used it so my vscode would stop marking the whole file as wrong.

Reply:
Should be removed now.